Successfully updated:
Created Frame node GHaBhq_Pl
Created Frame node auzAO74gd
Created Text node amf_4FF0u
Created Text node aZz7KZkAi
Created Frame node wminU5jB6
Updated attributes for node GHaBhq_Pl
Updated attributes for node auzAO74gd
Updated text for node amf_4FF0u
Updated attributes for node amf_4FF0u
Updated text for node aZz7KZkAi
Updated attributes for node aZz7KZkAi
Updated attributes for node wminU5jB6
Reordered node auzAO74gd within parent GHaBhq_Pl to index 0

XML Changes:
Index: node.xml
===================================================================
--- node.xml	Before
+++ node.xml	After
@@ -3881,20 +3881,73 @@
       width="fit-content"
       height="fit-content"
       top="0px"
       left="0px"
       font="Inter"
       textTruncation="2"
   >
     Truncated Text that is long enough to be truncated
   </TruncatedTextThatIsLongEnoughToBeTruncated>
 </Frame>
 <Frame
     nodeId="yKtdVfGM5"
     position="absolute"
     width="100px"
     height="100px"
     top="0px"
     left="0px"
     <!-- background color string or project color style path (if starts with /) -->
     backgroundColor="rgb(200, 200, 200)"
  />
+<Stack
+    nodeId="GHaBhq_Pl"
+    position="absolute"
+    width="400px"
+    height="300px"
+    top="0px"
+    left="0px"
+    <!-- background color string or project color style path (if starts with /) -->
+    backgroundColor="rgb(240, 240, 240)"
+    layout="stack"
+    gap="16px"
+    padding="20px"
+    stackDirection="vertical"
+    stackDistribution="center"
+    stackAlignment="center"
+>
+  <Frame
+      nodeId="auzAO74gd"
+      width="100%"
+      height="60px"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(100, 150, 200)"
+      borderRadius="8px"
+  >
+    <HeaderText
+        nodeId="amf_4FF0u"
+        position="absolute"
+        width="fit-content"
+        height="fit-content"
+        top="0px"
+        left="0px"
+        font="Inter"
+    >
+      Header Text
+    </HeaderText>
+  </Frame>
+  <BodyContentGoesHere
+      nodeId="aZz7KZkAi"
+      width="fit-content"
+      height="fit-content"
+      font="Inter"
+  >
+    Body content goes here
+  </BodyContentGoesHere>
+  <Frame
+      nodeId="wminU5jB6"
+      width="100%"
+      height="1fr"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(255, 255, 255)"
+      borderRadius="4px"
+   />
+</Stack>


Note: Set zoomIntoView=false if you want to use Framer app while MCP is working.